### ISSUE TITLE
Command line tools should put logging output to sys.stderr

### ISSUE THREAD

**Issue by zachriggle** [Gallopsled/pwntools#316]
This seems like it'd be a useful diversion from the current way of doing things.

It's useful when things break in the middle of a pipeline.

```
$ asm lol
[ERROR] There was an error running ...
$ asm lol -f raw | xxd
<hex encoded error message>
```

Ideally, this would happen

```
$ asm lol -f raw | xxd
[ERROR] There was an error running ...
```

**Comment by br0ns** [comment]
Why not let everything output by the logging module go to stderr?

**Comment by br0ns** [comment]
Fixed by #365.  Closing.
